otel.go

```diff
@@ -16,19 +16,15 @@ import (
 
 var closers []closerOp
 
-func LoadTraceProvider(ctx context.Context, useInsecure bool, collectorEndpoint string, sampleRatio float64) error {
+func LoadTraceProvider(ctx context.Context, collectorEndpoint string, sampleRatio float64) error {
 	if collectorEndpoint == "" {
 		return nil
 	}
 	r, err := serviceResource(ctx)
 	if err != nil {
 		return errors.Wrap(err, "making otel host resource")
 	}
-	var opts []otlptracegrpc.Option
-	if useInsecure {
-		opts = append(opts, otlptracegrpc.WithInsecure())
-	}
-	opts = append(opts, otlptracegrpc.WithEndpoint(collectorEndpoint))
+	opts := []otlptracegrpc.Option{otlptracegrpc.WithEndpoint(collectorEndpoint)}
```

Comment:
it looks like there's only one opt now. Could it make sense to just pass it to `otlptracegrpc.NewClient`?